zipline/finance/blotter.py

```diff
@@ -114,6 +114,9 @@ def order(self, sid, amount, style, order_id=None):
 
         return order.id
 
+    def order_batch(self, orders):
```

Comment:
`orders` here makes me think this is an iterable of `Order` objects.  Not sure what's a bettter name though, since `order_args` makes me think it's a single tuple of arguments, and `order_argses` just sounds like hobbit-speak.